ssl/ssl3.h

```diff
@@ -627,6 +627,9 @@ typedef struct ssl3_state_st {
      */
     unsigned char *alpn_selected;
     unsigned alpn_selected_len;
+    unsigned char *alpn_proposed;
```

Comment:
Hmmm... and I'm not sure that trying to stuff things following odd-sized chars are going to work either, given unknown packing by compilers and possible 16-bit?